source/end-user/integration/terraform.rst

```diff
@@ -1,3 +1,156 @@
 =========
 Terraform
 =========
+
+OpenStack Provider
+==================
+
+* https://www.terraform.io/docs/providers/openstack/index.html
+
+* Create a ``modules.tf`` file
+
+.. code-block:: json
+
+   provider "openstack" {
+     version = "~> 1.6"
+   }
+
+   terraform {
+     required_version = "= 0.11.7"
+   }
+
+   module "sample" {
+     source = "modules/sample"
+   }
+
+.. note::
+
+   ``provider:`` Before we can use the OpenStack provider, we configured the proper credentials.
+   Download the ``OpenStack RC File v3/v2`` from the horizon dashboard and evaluate the file with 
+   the command ``source``. Enter your OpenStack Password for youre project.
```

Comment:
Also document how to be able to not use the openrc file (https://www.terraform.io/docs/providers/openstack/index.html#example-usage). At least the use of clouds.yml file should be documented as well.